tokens/src/color/narmi.json

```diff
@@ -75,6 +75,7 @@
       "grey": { "value": "#4C4C4C" },
       "mediumGrey": { "value": "#8C8C8C" },
       "lightGrey": { "value": "#D9D9D9" },
+      "smokeGrey": { "value": "#F3F3F3" },
```

Comment:
The token `--bgColor-smokeGrey` already exists, but I am now seeing that it's confusing to use a `color` prefix for some tokens and a `bgColor` prefix for others.

I might copy more of those `bgColor` tokens into the `color` category...

Reply:
This line actually introduces this warning  while building
```
cssRgbColors
⚠️ /Users/reshavabraham/work/narmi/design_system/dist/tokens/css/rgbColors.css
While building rgbColors.css, token collisions were found; output may be unexpected.
    Output name rgb-smokeGrey was generated by:
        color.background.smokeGrey   243, 243, 243
        color.narmi.smokeGrey   243, 243, 243
This many-to-one issue is usually caused by some combination of:
    * conflicting or similar paths/names in property definitions
    * platform transforms/transformGroups affecting names, especially when removing specificity
    * overly inclusive file filters
```
So I've gone with `--bgColor-smokeGrey`